vllm/model_executor/models/interfaces.py

```diff
@@ -441,3 +442,16 @@ def supports_cross_encoding(
     model: Union[Type[object], object],
 ) -> Union[TypeIs[Type[SupportsCrossEncoding]], TypeIs[SupportsCrossEncoding]]:
     return is_pooling_model(model) and _supports_cross_encoding(model)
+
+
+class SupportsQuant:
```

Comment:
why does it only apply to these two models?

Reply:
> Follow ups
> * Add SupportsQuant mixin to all models which support quantization

This PR just introduces the mixin, separate PRs will be dedicated to integrating the mixin with model-specific tests